backend/.env.example

```diff
@@ -1,2 +1,5 @@
 PORT=5000
-DB_URI='mongodb://localhost:27017/your-database-name'
\ No newline at end of file
+DB_URI='mongodb://localhost:27017/your-database-name'
+BREVO_USERID='your-brevo-user-id'
+REVO_PASS='your-brevo-password'
```

Comment:
Thank you for adding this to .env.sample!
![nits](https://img.shields.io/badge/review-nits-green.svg) Typo found: 
x  REVO_PASS
o BREVO_PASS

Reply:
Thank you for catching the typo. I have just made the correction